14. Longest Common Prefix/code.cpp

```diff
@@ -0,0 +1,43 @@
+//https://leetcode.com/problems/longest-common-prefix/
+
+#include <iostream>
+#include <vector>
+
+std::string longestCommonPrefix(const std::vector<std::string>& strs) {
```

Comment:
strs is passed correctly, but check all functions with vectors and strings again